app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt

```diff
@@ -392,11 +394,30 @@ class PlayerFragment : Fragment(R.layout.fragment_player), OnlinePlayerOptions {
                 return@registerForActivityResult
             }
 
-            context?.contentResolver?.openOutputStream(uri)?.use { outputStream ->
-                screenshotBitmap?.compress(Bitmap.CompressFormat.PNG, 100, outputStream)
-            }
+            CoroutineScope(Dispatchers.IO).launch{
+                context?.contentResolver?.openOutputStream(uri)?.use { outputStream ->
+                    screenshotBitmap?.compress(Bitmap.CompressFormat.PNG, 100, outputStream)
+                }
+                screenshotBitmap = null
 
-            screenshotBitmap = null
+                withContext(Dispatchers.Main) {
+                    Snackbar.make(
+                        requireView(),
+                        R.string.screenshot_is_saved,
+                        Snackbar.LENGTH_LONG
+                    ).apply {
+                        setDuration(2500)
```

Comment:
```suggestion
                        2500
                    ).apply {
```

The duration can be set directly.